Firmware/ultralcd.cpp

```diff
@@ -4562,25 +4544,26 @@ static void lcd_calibration_menu()
 #endif //TMC2130
     MENU_ITEM_FUNCTION_P(_i("Selftest"), lcd_selftest_v);////MSG_SELFTEST c=18
     // MK2
-    MENU_ITEM_FUNCTION_P(_i("Calibrate XYZ"), lcd_mesh_calibration);////MSG_CALIBRATE_BED c=18
+    MENU_ITEM_GCODE_P(_i("Calibrate XYZ"), PSTR("M45"));////MSG_CALIBRATE_BED c=18
     // "Calibrate Z" with storing the reference values to EEPROM.
-    MENU_ITEM_SUBMENU_P(_T(MSG_HOMEYZ), lcd_mesh_calibration_z);
+    MENU_ITEM_GCODE_P(_T(MSG_HOMEYZ), PSTR("M45 Z"));
```

Comment:
unintended behaviour change. The code change makes it so clicking these menu items doesn't go back to the status screen. It also makes it possible to spam click this